cse-lib/network-processing/pom.xml

```diff
@@ -13,6 +13,34 @@
     <artifactId>gridcapa-cse-network-processing</artifactId>
 
     <dependencies>
+        <dependency>
+            <groupId>com.farao-community.farao</groupId>
+            <artifactId>farao-crac-creation-util</artifactId>
+            <version>${farao.core.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>com.farao-community.farao</groupId>
+            <artifactId>farao-crac-creator-cse</artifactId>
+            <version>${farao.core.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>com.powsybl</groupId>
+            <artifactId>powsybl-ucte-converter</artifactId>
+        </dependency>
+
+        <!-- Runtime dependencies -->
+        <dependency>
+            <groupId>com.powsybl</groupId>
+            <artifactId>powsybl-iidm-impl</artifactId>
+            <scope>runtime</scope>
+        </dependency>
```

Comment:
I think you don't need runtime dependency here, the client application would load it